.github/actions/unit_tests/action.yml

```diff
@@ -19,6 +19,6 @@ runs:
       # Note: Each of these test commands need to be exposed on the monorepo
       # root and *also* on the PWA package. This section is run on both.
 
-      # Always run fast unit tests
-      npm run test
+      # Always run fast unit tests - CI is flaky, so let's try three times!
+      npm run test || npm run test || npm run test
```

Comment:
Dumb question, are they running in sequence or parallel?

Reply:
`a || b` executes `a`, and then only executes `b` if `a` exited with a non-zero code. So this runs unit tests, tries again if it fails, then tries again if it fails again.